core/test/utils_test.jl

```diff
@@ -116,19 +131,19 @@ end
 end
 
 @testitem "Expand logic_mapping" begin
-    logic_mapping = Dict{Tuple{Int, String}, String}()
-    logic_mapping[(1, "*T*")] = "foo"
-    logic_mapping[(2, "FF")] = "bar"
+    logic_mapping = Dict{Tuple{Ribasim.NodeID, String}, String}()
+    logic_mapping[(Ribasim.NodeID(1), "*T*")] = "foo"
+    logic_mapping[(Ribasim.NodeID(2), "FF")] = "bar"
     logic_mapping_expanded = Ribasim.expand_logic_mapping(logic_mapping)
 
-    @test logic_mapping_expanded[(1, "TTT")] == "foo"
-    @test logic_mapping_expanded[(1, "FTT")] == "foo"
-    @test logic_mapping_expanded[(1, "TTF")] == "foo"
-    @test logic_mapping_expanded[(1, "FTF")] == "foo"
-    @test logic_mapping_expanded[(2, "FF")] == "bar"
+    @test logic_mapping_expanded[(Ribasim.NodeID(1), "TTT")] == "foo"
```

Comment:
If you get tired of typing `Ribasim.NodeID`, you can also add `using Ribasim: NodeID` at the top of the testitem, so you can just use `NodeID` directly.

Note that: By default, Test and the package being tested will be imported into the testitem automatically.